partcad-cli/src/partcad_cli/click/commands/adhoc/convert.py

```
@@ -0,0 +1,80 @@
+@click.command(help="Convert CAD files between formats")
+@click.option(
+    "--input", "input_type",
+    type=click.Choice(
+      [
+         "step",
+         "brep",
+         "stl",
+         "3mf",
+         "scad",
+         "cadquery",
+         "build123d"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)